src/Command/ProbeDispatcherCommand.php

```diff
@@ -156,7 +156,7 @@ private function setUp(InputInterface $input)
             $this->queues[$i] = new Queue(
                 $this->workerManager,
                 $i,
-                getenv('SLAVE_NAME'),
+                $_ENV['SLAVE_NAME'],
```

Comment:
This has the same issue as described in the review of `src/AlertDestination/Mail.php`.